app/graphql/types/invoices/object.rb

```diff
@@ -46,6 +46,18 @@ class Object < Types::BaseObject
       field :fees, [Types::Fees::Object], null: true
       field :invoice_subscriptions, [Types::InvoiceSubscription::Object]
       field :subscriptions, [Types::Subscriptions::Object]
+
+      # NOTE(legacy): Remove with taxes feature
```

Comment:
Why do we need to keep these fields as legacy on graphql? 🤔

Reply:
@rsempe it's only temporary to be able to merge it quickly without breaking the front-end and then simply to a new PR with all those fields removed at once. My idea was to prevent a "ping-pong effect" as this feature changes many thinks on GraphQL side.